src/test/java/cc/robotdreams/WebTest2.java

```diff
@@ -0,0 +1,53 @@
+package cc.robotdreams;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.w3c.dom.Element;
+import org.w3c.dom.Text;
+
+public class WebTest2
+{
+    WebDriver driver;
+    @BeforeTest
+    public void setup() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--start-maximized");
+
+        driver = new ChromeDriver(options);
+    }
+    @Test
+    public void openWebsite() throws InterruptedException {
+        driver.get("https://demoqa.com/webtables");
+
+        driver.findElement(By.xpath("//div/button[@id='addNewRecordButton']")).click();
+        driver.findElement(By.xpath("//input[@id='firstName']")).sendKeys("Карл");
+        driver.findElement(By.xpath("//input[@id='lastName']")).sendKeys("Маркс");
+        driver.findElement(By.xpath("//input[@id='userEmail']")).sendKeys("carl@gmail.com");
+        driver.findElement(By.xpath("//input[@id='age']")).sendKeys("21");
+        driver.findElement(By.xpath("//input[@id='salary']")).sendKeys("2500");
+        driver.findElement(By.xpath("//input[@id='department']")).sendKeys("QA");
+        driver.findElement(By.xpath("//div/button[@id='submit']")).click();
+        WebElement elementToCheck = driver.findElement(By.xpath(
+                "//*[@id=\"app\"]/div/div/div[2]/div[2]/div[2]/div[3]/div[1]/div[2]/div[4]"));
```

Comment:
Very bad locator. You can use `//div[@role='rowgroup'][4]` instead but even this locator is not good for next checks.
Read this code and try to understand what it does:
```
SoftAssert softAssert = new SoftAssert();
// Get all emails from table
List<WebElement> emails = webDriver.findElements(By.xpath("//div[@role='rowgroup']//div[@role='gridcell' and text()][4]"));
// Find row number by email
int rowNumber = -1;
for (int i = 0; i < emails.size(); i ++)
   if ("carl@gmail.com".equalsIgnoreCase(emails.get(i).getText())) {
      rowNumber = i + 1;
      break;
   }
Assert.assertNotEquals(rowNumber, -1, "Record not found by email");
// Get element with first name
WebElement firstName = webDriver.findElement(By.xpath("//div[@role='rowgroup'][" + rowNumber  + "]//div[@role='gridcell'][1]"));
// Check first name value
softAssert.assertEquals(firstName.getText(), "Карл", "Saved first name is not correct...");
...
softAssert.assertAll();
```